### DISCUSSION TITLE
🚀 Enhancement: Support SQLite

### DISCUSSION THREAD

**Discussion by soranoo** [appwrite/appwrite#11527] [Ideas]
### 🔖 Enhancement description

Add SQLite as an option in db selection. (I would to see appwrite either support D1 or Turso.)

### 🎤 Pitch

Edge computing is becoming increasingly popular in database management, with notable edge databases like Cloudflare D1 and Turso leveraging SQLite for global distribution. Supporting SQLite would align with this trend and enhance the ability to scale and distribute data effectively across edge networks.

I saw [utopia-php/database](https://github.com/utopia-php/database/) support SQLlite but I don't see any appwrite docs talking about SQLite.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@soranoo, thanks for creating this issue! 🙏🏼 We definitely want to allow using different underlying databases. For now, we have a lot of improvements planned for Appwrite Databases so it might be better to only officially support one until things settle down a bit more.
